examples/darwin-framework-tool/commands/tests/TestCommandBridge.h

```diff
@@ -101,16 +106,44 @@ class TestCommandBridge : public CHIPCommandBridge,
         SetCommandExitStatus(err);
     }
 
-    /////////// DelayCommands Interface /////////
-    void OnWaitForMs() override
+    /////////// DelayCommands /////////
+    CHIP_ERROR WaitForMs(
+        const char * _Nullable identity, const chip::app::Clusters::DelayCommands::Commands::WaitForMs::Type & value)
     {
+        [NSThread sleepForTimeInterval:value.ms / 1000];
```

Comment:
This does not look right.  `WaitForMs` needs to be an async wait, during which other messsages can be processed and whatnot, no?

Reply:
Yes, you're correct. Let me update this to do NextTest() in a dispatch_after() call.

Reply:
Updated.